.github/workflows/browser-compatibility-test-previous-major-version.yml

```diff
@@ -0,0 +1,308 @@
+name: Previous Major Version Browser Compatibility Test
+
+on:
+  schedule:
+    # More information on cron https://crontab.guru/
+    # GitHub actions is using UTC time. Scheduling action at 12 am PST
```

Comment:
Doesn't seem like the time is 12am? Also I thought we would start this at night and not in the morning?

Reply:
Gotcha.